### ISSUE TITLE
ReadME documentation on batch

### ISSUE THREAD

**Issue by RyanMarten** [bespokelabsai/curator#48]
Started some notes below, can formally write them up and benchmark

TLDR; for OpenAI batch is slow but cheap (at larger scales 10M+ might be faster?)

Right now we only support batch for OpenAI, although we can add Claude (which now has it). 
Completion time is dependent on OpenAI's current load and can take up to 24 hours (and can technically "expire" if it is not done by then, although I haven't seen this yet). 
Current batch size is default to 1k.  
Batch potentially could be faster for very very large jobs (>10M) since the throughput (theoretically?) can exceed 500 QPS (which is what online maxes out for gpt-4o-mini due to rate limits), but I haven't benchmarked that (since it is expensive and would only do it for an actual run)

Therefore the suggested workflow:
you can do online (batch=False, this is the default value) for small scale / test for larger scale (1k-10k) 
for medium scale (100k+) if you want to save money, just use batch=True and run it overnight, or just run without if you want it quickly
for large scale (1M+) probably should only do batch=True unless you are really desperate and in a rush and willing to pay $$$
for xlarge scale (10M+) we need to benchmark and see if you can actually get faster speeds with batch=True
